### ISSUE TITLE
Implement command to get cpu usage benchmark

### ISSUE THREAD

**Issue by govargo** [kubernetes/minikube#11077]
/kind feature

Currently, minikube has several driver choices.
The drivers have each characteristics and they affect local machine's performance differently.

Of course, low CPU load and high performance are desirable, but it is important to know each characteristics and performance values ​​of each drivers.

In the past, Thomas took cpu usage benchmark using [cstat](https://github.com/tstromberg/cstat) on September 2020.
[Local Kubernetes - CPU Busy% overhead on macOS](https://docs.google.com/spreadsheets/d/12mBmRmNL-31fOQLjm4v0AvGc0iGwBrEHbQ7Uo_ysCnQ)
This benchmark compared minikube's drivers and similer products(Kind, K3d, Docker for Desktop).

Now if we'd like to take cpu usage benchmark, It takes long time and to summarize manually.
So, in this issue, I'll aim to implement the `make XXXX` command to do to reduce mannual costs.

**Comment by govargo** [comment]
I took latest cpu usage benchmark in minikube v1.19.0(beta) using `cstat`.
[Local Kubernetes - CPU Busy% overhead on macOS & Debian - April 2021](https://docs.google.com/spreadsheets/d/1k3JYvf7BlrBhImk7SFfrdXRg4E70PyqYKn_l27L2ouc)

It compares minikube's driver and Docker for Mac, Kind, K3s on macOS.
And it also compares on Linux Debian.
![Screenshot 2021-04-12 10 33 06 PM](https://user-images.githubusercontent.com/44902466/114402746-1a870280-9bdf-11eb-8db6-ba58f359bdb1.png)

In macOS, the vm driver(virtualbox, hyperkit) is lower cpu usage than Docker something(KIC, Kind).
I think that Docker for Mac build Linux of VM and it runs Docker engine on VM. It causes higher load.

![Screenshot 2021-04-12 10 33 58 PM](https://user-images.githubusercontent.com/44902466/114403278-9c772b80-9bdf-11eb-97f5-f85674862d47.png)

On the other hands, interestingly, the vm driver(virtualbox, kvm2) is higher cpu usage than Docker something(KIC, Kind) in Linux.

In other words, the results are contrast when comparing Linux and macOS.
